Add actionable items to the issue_template (#44)
aicoe-aiops/operate-first-data-science-community: .github/ISSUE_TEMPLATE/topic_submission.md

Any information you'd like to provide about yourself, the presenter. This may including name, github username, organization, or project affiliations, etc. 

**Additional Info**
If applicable, please provide links to any relevant blogs, videos, git repos, etc relating to the suggested presentation. 

> **Actionable Items**

Reviewer: For an external contributor trying to propose their topic these Action Items may not be relevant. For example they might contribute their approved for finalized topic to the agenda document however the rest of the items seem like the organizers' responsibilities.

Should we state that here? Maybe like
```suggestion
**Actionable Items** (Following actions would be carried out by the Organizers of the Operate First Data Science Community Meetup)
```

Author: Sounds good :+1:

- [ ]  Add the topic and description to the [agenda](../../docs/content.md).

> - [ ] Send an invite via email using the [email template](../../docs/email_templates/invite_template.md) and Operate First twitter by adding a PR to [operate-first-twitter](https://github.com/operate-first/operate-first-twitter).

Reviewer: ```suggestion
- [ ] Send an invite via email using the [email template](../../docs/email_templates/invite_template.md) and a tweet through the Operate First twitter Account by adding a PR to [operate-first-twitter](https://github.com/operate-first/operate-first-twitter).
```

- [ ] Present the talk in Operate First Data Science Meetup. Follow the [format](../../docs/format.md) and record the meetup.
- [ ] Edit the video recording of the Meetup and upload the video recording to [Operate First YouTube channel](https://www.youtube.com/channel/UCe87bwqlGoBQs2RvMQZ5_sg).

> - [ ] Send out an update via email using the [email template](../../docs/email_templates/postmeetup_update.md) and Operate First twitter by adding a PR to [operate-first-twitter](https://github.com/operate-first/operate-first-twitter).   

Reviewer: ```suggestion
- [ ] Send out meetup recording via email using the [email template](../../docs/email_templates/postmeetup_update.md) and tweet through the Operate First twitter account by adding a PR to [operate-first-twitter](https://github.com/operate-first/operate-first-twitter).   
```
